analytics_data_api/v0/tests/views/__init__.py

```diff
@@ -100,15 +100,28 @@ class APIListViewTestMixin(object):
     list_name = 'list'
     default_ids = []
     always_exclude = ['created']
+    test_post_method = False
 
-    def path(self, ids=None, fields=None, exclude=None, **kwargs):
-        query_params = {}
-        for query_arg, data in zip([self.ids_param, 'fields', 'exclude'], [ids, fields, exclude]) + kwargs.items():
-            if data:
-                query_params[query_arg] = ','.join(data)
-        query_string = '?{}'.format(urlencode(query_params))
+    def path(self, query_data={}):
+        concat_query_data = {param: ','.join(arg) for param, arg in query_data.items() if arg}
+        query_string = '?{}'.format(urlencode(concat_query_data)) if concat_query_data else ''
         return '/api/v0/{}/{}'.format(self.list_name, query_string)
 
+    def validated_request(self, ids=None, fields=None, exclude=None, **extra_args):
```

Comment:
This is the function I'm most interested in feedback on. I'm making it so every request is done as both a GET and POST, and the results are compared before returning. Let me know what you think @efischer19.